estate/models/estate_property_offer.py

```diff
@@ -10,5 +12,22 @@ class EstatePropertyOffer(models.Model):
         selection=[('accepted', "Accepted"), ('rejected', "Rejected")],
         copy=False,
     )
-    partner_id = fields.Many2one("res.partner", string="Partner ID", required=True)
+    partner_id = fields.Many2one('res.partner', string="Partner ID", required=True)
     property_id = fields.Many2one('estate.property', string="Property", required=True)
+    validity = fields.Integer(string="Validity", default=7)
+    date_deadline = fields.Date(string="Date Deadline", compute='_compute_date_deadline', inverse='_inverse_date_deadline')
+
+    @api.depends('validity')
+    @api.depends('create_date')
+    def _compute_date_deadline(self):
+        for record in self:
+            if record.create_date is False:
```

Comment:
In Python we try to avoid comparisons to `True`, `False`, `None`... unless you'd actually really need to distinguish `False` from `None`.
So, here if you swap both branches of the `if` you can simply write `if record.create_date:`.